libraries/bootstrap.php

```diff
@@ -10,7 +10,12 @@
 
 defined('_JEXEC') or die;
 
-// Set the platform root path as a constant if necessary.
+/**
+ * Set the platform root path as a constant if necessary.
+ *
+ * @deprecated __DEPLOY_VERSION__ will be remove in 6.0
```

Comment:
```suggestion
 * @deprecated __DEPLOY_VERSION__ will be removed in 6.0
```
Grammar fix. Do the same at all other places with the same error.